assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js

```diff
@@ -17,22 +17,93 @@
 /**
  * WordPress dependencies
  */
+import { __ } from '@wordpress/i18n';
 import { Fragment } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
+import Data from 'googlesitekit-data';
 import Header from '../../../../components/Header';
 import HelpMenu from '../../../../components/help/HelpMenu';
+import Layout from '../../../../components/layout/Layout';
+import { Cell, Grid, Row } from '../../../../material-components';
+import PageHeader from '../../../../components/PageHeader';
+import AdBlockingSetupSVG from '../../../../../svg/graphics/ad-blocking-recovery-setup.svg';
+import Link from '../../../../components/Link';
+import {
+	BREAKPOINT_SMALL,
+	useBreakpoint,
+} from '../../../../hooks/useBreakpoint';
+import { CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+const { useSelect } = Data;
 
 export default function AdBlockingRecoveryApp() {
+	const breakpoint = useBreakpoint();
+
+	const settingsURL = useSelect(
+		( select ) =>
+			`${ select( CORE_SITE ).getAdminURL(
+				'googlesitekit-settings'
+			) }#/connected-services/adsense`
+	);
+
+	const isTabletWidthOrLarger = breakpoint !== BREAKPOINT_SMALL;
+
 	return (
 		<Fragment>
 			<Header>
 				<HelpMenu />
 			</Header>
 			<div className="googlesitekit-ad-blocking-recovery">
-				Ad Blocking Recovery
+				<div className="googlesitekit-module-page">
+					<Grid>
```

Comment:
Each grid should have at least one row and cell in it. Maybe we don't need the `Grid` element here?

Reply:
That's probably true, I'll try it out.

Reply:
Looks like that base `<Grid>` is used to apply the baseline margin to the plugin. We'll need to keep it in here 😓

with: 
<img width="1253" alt="CleanShot 2023-05-31 at 15 12 58" src="https://github.com/google/site-kit-wp/assets/90871/fb740086-0c36-49ac-8e9a-16fe3c0d9cc7">

without:
<img width="1260" alt="CleanShot 2023-05-31 at 15 12 53" src="https://github.com/google/site-kit-wp/assets/90871/cae78d04-1b89-4560-932e-16408c302828">